src/backend/drm/surface/gbm.rs

```diff
@@ -180,17 +177,26 @@ where
                                                   // It has no further use.
         };
 
-        let fb = match attach_framebuffer(&drm, &buffer) {
+        let fb = match framebuffer_from_bo(drm.device_fd(), &buffer, true) {
             Ok(fb) => fb,
-            Err(err) => return Err((swapchain.allocator, err)),
+            Err(err) => {
+                return Err((
+                    swapchain.allocator,
+                    Error::DrmError(DrmError::Access {
```

Comment:
This appears at least twice, maybe a From implementation could make things neater.

Reply:
Done